app/queues/dns/update-dns-record-flow.server.ts

```diff
@@ -34,6 +36,8 @@ export const updateDnsRequest = async (data: UpdateDnsRequestData) => {
     course: data.course,
     ports: data.ports,
   });
+  // And set its status to pending
+  await updateDnsRecordStatus(id, DnsRecordStatus.pending);
```

Comment:
I think we should also update the DB after Route53 gets updated. I will write a PR for this one.